scoper.py

```diff
@@ -334,7 +331,8 @@ def main(self, youtube_video_url, limit=5, languages=None, mode='FUZZY'):
         for timestamp in timestamps:
             captions_extracted.append(timestamps_and_captions[timestamp])
 
-        pretty_print(list(zip(captions_extracted, normalize_time(timestamps))))
+        # pretty_print(list(zip(captions_extracted, normalize_time(timestamps))))

```

Comment:
Also, I think it's better if to uncomment this line as commenting it mutes the CLI output, so if someone were not using the web GUI, they wouldn't be seeing any output.